fivetran_provider_async/hooks.py

```diff
@@ -92,7 +95,7 @@ async def get_connector_async(self, connector_id):
         resp = await self._do_api_call_async(("GET", endpoint))
         return resp["data"]
 
-    async def get_sync_status_async(self, connector_id, previous_completed_at):
+    async def get_sync_status_async(self, connector_id, previous_completed_at, reschedule_time=0):
```

Comment:
```suggestion
    async def get_sync_status_async(self, connector_id, previous_completed_at, reschedule_wait_time=0):
```

Reply:
Since it is the wait time before reschedule, would make sense to call it that way for readability

Reply:
Changed it here https://github.com/astronomer/airflow-provider-fivetran-async/pull/25/commits/84e8d98914e4eb2e4673f7ee9709bdd1ff268cce